README.md

```diff
@@ -20,4 +20,18 @@ We are showing how Commands are being used side by side by MVVMLight, Caliburn.M
 We are showing how Messaging Pattern (AKA EventAggregation) is being used side by side by MVVMLight, Caliburn.Micro and of course ReactiveUI.
```

Comment:
I would remove the sections above your change for the moment. They are based on the old-old samples.

Reply:
Ok, old sections removed